rightscript.go

```diff
@@ -893,3 +933,38 @@ func (rs *RightScript) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 	return nil
 }
+
+func getRightScriptNameFromInput(href string) (string, error) {
+	if isDirectory(href) {
```

Comment:
Don't we want to be checking that it is a regular file (or at least points to one) rather than a directory?